containers/proxy-httpd-image/Dockerfile

```
@@ -1,25 +1,29 @@
 #!BuildTag: uyuni/proxy-httpd:latest uyuni/proxy-httpd:%PKG_VERSION% uyuni/proxy-httpd:%PKG_VERSION%.%RELEASE% uyuni/proxy-httpd:beta
 
-ARG BASE=opensuse/leap:15.3
+ARG BASE=registry.suse.com/bci/bci-base:15.3
 FROM $BASE
 
 
 ARG PRODUCT_REPO
+ARG PRODUCT=Uyuni
+ARG VENDOR="Uyuni project"
+ARG URL="https://www.uyuni-project.org/"
+ARG REFERENCE_PREFIX="registry.opensuse.org/uyuni"
```

Are you sure this will be the prefix? Is this something that can be forced by using the reference?

For CI the images use `registry.opensuse.org/systemsmanagement/uyuni/master/docker/containers/<image_name>`

So unless we can force the prefix, this should be 
```suggestion
ARG REFERENCE_PREFIX="registry.opensuse.org/systemsmanagement/uyuni/stable/containers/uyuni"
```
Worst case we can merge this PR and test it.

But even if we can merge the prefix: will this mean that master images will overwrite stable images?